unit-tests-jest/ts/numbers.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers: number[]): number[] {
+  return numbers.filter((n) => (n + 1) % 2);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ */
+export function toDollars(amount: number): string {
+  return `${amount < 0 ? '-' : ''}$${Math.abs(amount).toFixed(2)}`;
+}
+
+/**
+ * Returns a new array of numbers where every entry has been divided by the
+ * given divisor. Does not modify the original array.
+ */
+export function divideBy(numbers: number[], divisor: number): number[] {
+  const newNumbers: number[] = [];
+  for (let i = 0; i < numbers.length; i++) {
+    newNumbers.push(Math.round((numbers[i] / divisor) * 100000) / 100000);
+  }
+  return newNumbers;
+}
+
+/**
+ * Modifies an object by multiplying the value of each key, but only if that
+ * value is a number. Key names are arbitrary.
+ */
+export function multiplyBy(
+  obj: Record<string, unknown>,
+  multiplier: number
+): Record<string, unknown> {
+  Object.entries(obj).forEach(([key, value]) => {
+    if (typeof value === 'number') {
+      obj[key] = Number(value) * multiplier;
```

Comment:
In this block you know `value` is a number, so there's no reason to convert it with `Number()`.